src/style.css

```diff
@@ -0,0 +1,53 @@
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-around;
+  width: 100px;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star:nth-of-type(2),
+.stars--1 .stars__star:nth-of-type(3),
+.stars--1 .stars__star:nth-of-type(4),
+.stars--1 .stars__star:nth-of-type(5),
+
+.stars--2 .stars__star:nth-of-type(3),
+.stars--2 .stars__star:nth-of-type(4),
+.stars--2 .stars__star:nth-of-type(5),
+
+.stars--3 .stars__star:nth-of-type(4),
+.stars--3 .stars__star:nth-of-type(5),
+
+.stars--4 .stars__star:nth-of-type(5) {
+  --size: 16px;
+
+  background: url(./images/star.svg) left no-repeat;
+  width: var(--size);
```

Comment:
too much duplications, set `width` and `height` to the `stars__star` class, and change `background-image` like below 
`.stars--1 > :nth-child(1),
.stars--2 > :nth-child(-n + 2),`